### PR TITLE
[New Tutorial] S3 Batch operations

### PR DESCRIPTION
**Pull request by dsonger73**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE). Notwithstanding the above, if you provide source code text within your content piece, you are providing such source code under the MIT-0 license.**

### CODE REVIEW

**Comment by scribwell** [comment]
@DalM2023 This tutorial is ready for editorial review. At a quick glance, I noticed a few things that I want to make sure you include in your edits. The authors are new to MarkDown so you'll find some formatting issues you'll need to highlight for them to fix, including that the steps are all misnumbered. Also, they accidentally put the intro paragraph into the subhead instead of in the body of the text so you'll need to call out that they fix that. Thank you! (FYI @damonk10)

File: tutorials/s3-batch-operations-restore/index.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+---
+title: "Using Amazon S3 Batch Operations to restore archived objects from an Amazon S3 Glacier storage classes"
```
Comment (DalM2023) [inline tutorials/s3-batch-operations-restore/index.md:2]:
```suggestion
title: "Use Amazon S3 Batch Operations To Restore Archived Objects From an Amazon S3 Glacier Storage Classes"

```

File: tutorials/s3-batch-operations-restore/index.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
+---
```
Comment (DalM2023) [inline tutorials/s3-batch-operations-restore/index.md:1]: Hello!

I made editorial suggestions throughout the piece. My lighter edits focus primarily on correcting titles to AP Style and grammatical errors and from the requirements based on the Community.AWS Editorial Review Process. I flagged some issues with spacing and structure too. 

Please see my notes below for more information. 

Overall, well done!!

**Review by DalM2023 — COMMENTED** [review]
1st round edit done. Ready for review.

File: tutorials/s3-batch-operations-restore/index.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+| ToC |
+|-----|
+
+
+<!-- Use the above to auto-generate the table of content. Only build out a manual one if there are too many (sub) sections. -->
+
+
+## Overview
+
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:37]: moved paragraph as suggested

File: tutorials/s3-batch-operations-restore/index.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+* **An AWS account:** If you don't already have an account, follow the [_Setting Up Your Environment_](https://aws.amazon.com/getting-started/guides/setup-environment/) getting started guide for a quick overview.
+
+## Implementation
+
+### Step 1: Create an Amazon S3 bucket
+
+ a. Log in to the [_AWS Management Console_](https://console.aws.amazon.com/) using your account information. In the search bar, enter **S3**, then select **S3** from the results.
+
+
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:67]: There are several spacing issues due to formatting from bringing this over from quip - hopefully I have corrected them all

File: tutorials/s3-batch-operations-restore/index.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+ 1. From the [_Amazon S3 console_](https://s3.console.aws.amazon.com/s3/home), search for the bucket that you created in Step 1, and select the bucket name.
+
+![Alt text](Images/image(2).png)
+
+1. Next, select the **Objects** tab. Then, from within the **Objects** section, choose **Upload**.
+
+![Alt text](Images/image(3).png)
+    1. Then, in the **Upload** section, choose **Add files**. Navigate to your local file system to locate the test file that you created above. Select the appropriate file, and then choose **Open**. Your file will be listed in the **Files and folders** section. 
+ ![Alt text](Images/image(4).png)       
+
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:102]: Corrected all the spacing issues I could find

File: tutorials/s3-batch-operations-restore/index.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
...ccessfully processes at least one object, Amazon S3 generates a completion report after the S3 Batch Operations job completes, fails, or is cancelled. You have the option to include all tasks or only failed tasks in the completion report. The completion report contains additional information for each task, including the object key name and version, status, error codes, and [_descriptions of any errors_](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-failure-codes.html). Completion reports provide an easy way to view the status of each object restored using the S3 Batch Operations job and identify failures, if any. In this example, we chose to **Generate completion report** for **All tasks** so that we can review the status of all objects within this job. Alternatively, you can also choose to view the status of objects that failed to restore only by choosing the **Failed tasks only** option. We have provided the destination bucket as the destination for the completion report. For additional examples of completion reports, see [_Examples: S3 Batch Operations completion reports_](https://docs.aws.amazon.com/AmazonS3/latest/userguide/batch-ops-examples-reports.html).
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: spacing correced - hopefully

File: tutorials/s3-batch-operations-restore/index.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+1. **View S3 Batch Operations completion reports**
+    a. S3 Batch Operations generates a report for jobs that have completed, failed, or cancelled. Select the path you’ve configured to save the completion reports.
+
+![Alt text](Images/image(27).png)
+    1. Download the completion report to analyze the status of each task.
+1. In the following example, all the objects have been successfully restored.
+2. The description of errors for each failed task can be used to diagnose issues that occur during job creation, such as permissions.
+
+![Alt text](Images/image(28).png)
+1. **Clean up resources**
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: added recommended sentence

File: tutorials/s3-batch-operations-restore/index.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+
+    a. Delete the IAM role and the IAM policy.
+1. If you have logged out of your AWS Management Console session, log back in. Navigate to the [_IAM console_](https://console.aws.amazon.com/iam/) and select **Roles** from the left menu options.
+2. Delete the role created for this tutorial.
+3. Navigate to **Policies** from the left menu options.
+4. Delete the IAM policy that was created for this tutorial.
+
+## Conclusion
+
+Congratulations! You have learned how to use S3 Batch Operations to restore archived objects. You can customize the restore tier and expiration time depending on your access needs. You can also use S3 Batch Operations to perform other types of requests such as replication, Lambda invocation, replacing object ACLs, tagging and enabling object lock.
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: Added recommmended statement  - may need a re-write

File: tutorials/s3-batch-operations-restore/index.md [hunk 9]
```diff
@@ -0,0 +1,239 @@
...
+
+## Next Steps
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: done

File: tutorials/s3-batch-operations-restore/index.md [hunk 10]
```diff
@@ -0,0 +1,239 @@
+---
+title: "Using Amazon S3 Batch Operations to restore archived objects from an Amazon S3 Glacier storage classes"
+description: "Imagine it’s 4 p.m. on a Friday and your legal department just tasked you with providing access to files that have been stored in an S3 Flexible Retrieval archive storage class over the last several years. There’s a tight deadline and there are hundreds of files that need to be restored in order for the legal team to complete their work. In this tutorial, we’ll show you how to quickly restore objects by configuring an S3 Batch Operations restore job, which streamlines the process of retrieving objects from archival storage classes ensuring swift access to vital data while adhering to cost-effective storage practices."
+tags:
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: added additional tags

File: tutorials/s3-batch-operations-restore/index.md [hunk 11]
```diff
@@ -0,0 +1,239 @@
...
+
+## Prerequisites
+
+Before starting this tutorial, you will need:
+
+* **An AWS account:** If you don't already have an account, follow the [_Setting Up Your Environment_](https://aws.amazon.com/getting-started/guides/setup-environment/) getting started guide for a quick overview.
+
+## Implementation
+
+### Step 1: Create an Amazon S3 bucket
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:63]: added steps to break appart the tutorial into logical steps

File: tutorials/s3-batch-operations-restore/index.md [hunk 12]
```diff
@@ -0,0 +1,239 @@
+---
+title: "Using Amazon S3 Batch Operations to restore archived objects from an Amazon S3 Glacier storage classes"
+description: "Imagine it’s 4 p.m. on a Friday and your legal department just tasked you with providing access to files that have been stored in an S3 Flexible Retrieval archive storage class over the last several years. There’s a tight deadline and there are hundreds of files that need to be restored in order for the legal team to complete their work. In this tutorial, we’ll show you how to quickly restore objects by configuring an S3 Batch Operations restore job, which streamlines the process of retrieving objects from archival storage classes ensuring swift access to vital data while adhering to cost-effective storage practices."
```
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: done - please check description to see if that meets the requirements

File: tutorials/s3-batch-operations-restore/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:37]: completed

**Review by cherbk — APPROVED** [review]

**Review by cherbk — APPROVED** [review]

File: tutorials/s3-batch-operations-restore/index.md [hunk 13]
```diff
@@ -0,0 +1,272 @@
+---
+title: "Use Amazon S3 Batch Operations to Restore Archived Objects From Amazon S3 Glacier Storage Classes"
```
Comment (damonk10) [inline tutorials/s3-batch-operations-restore/index.md:2]: Check this title. Is it from "Glacier Storage Classes" or "a Glacier Storage Class"?

File: tutorials/s3-batch-operations-restore/index.md [hunk 14]
```diff
@@ -0,0 +1,272 @@
...
+
+![Alt text](Images/image(21).png)
+
+Upon creation of the job, Batch Operations processes the manifest. If successful, it will change the job status to **Awaiting your confirmation to run**. You must confirm the details of the job and select **Run job**.
+
+![Alt text](Images/image(22).png)
+
+You should then see a notification of successful confirmation for the job displayed in the banner at the top of the Batch Operations page.
+
+![Alt text](Images/image(23).png)
```
Comment (damonk10) [inline tutorials/s3-batch-operations-restore/index.md:204]: Please add alt text for all the images. That should just be one line describing what's in the image, as though the reader is unable to see it themselves. Ask yourself: "What's the key piece of information I want this image to communicate to a reader?"

File: tutorials/s3-batch-operations-restore/index.md [hunk 15]
```diff
@@ -0,0 +1,272 @@
...
+3. Select the radio button to the left of the source bucket you created for this tutorial, and choose the **Empty** button.
+4. Select **Empty**. Review the warning message. If you desire to continue emptying this bucket, enter the bucket name into the Empty bucket confirmation box, and choose **Empty bucket.**
+
+**Note:** Objects that are archived to the S3 Glacier Flexible Retrieval storage class are charged for a minimum storage duration of 90 days. Objects deleted prior to the minimum storage duration incur a pro-rated charge equal to the storage charge for the remaining days. Objects that are deleted, overwritten, or transitioned to a different storage class before the minimum storage duration will incur the normal storage usage charge plus a pro-rated storage charge for the remainder of the minimum storage duration. For more information, refer to the [_S3 Pricing page_](https://aws.amazon.com/s3/pricing/).
+
+![Alt text](Images/image(29).png)
+
+1. Delete the bucket.
```
Comment (damonk10) [inline tutorials/s3-batch-operations-restore/index.md:250]: Please double check numbers throughout the piece. It looks like some of them have been split up. Here, there are two 1's. Make sure the sequence is what you want it to be in each instance.

File: tutorials/s3-batch-operations-restore/index.md (hunk 13 above)
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:2]: We used one storage class for the example in the tutorial but the process will apply to either of the Glacier storage classes.  Not sure from an editing standpoint how that should be worded.

File: tutorials/s3-batch-operations-restore/index.md (hunk 15 above)
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md:250]: corrected numbering

File: tutorials/s3-batch-operations-restore/index.md (hunk 14 above)
Comment (dsonger73) [reply tutorials/s3-batch-operations-restore/index.md]: text has been added to all images

**Review by damonk10 — APPROVED** [review]

**Pull request merged**